ravenframework/Samplers/EnsembleForward.py

```diff
@@ -18,30 +18,29 @@
   @author: alfoa
   supercedes Samplers.py from alfoa
 """
-#for future compatibility with Python 3--------------------------------------------------------------
+# for future compatibility with Python 3------------------------------------------------------------
 from __future__ import division, print_function, unicode_literals, absolute_import
-#End compatibility block for Python 3----------------------------------------------------------------
+# End compatibility block for Python 3--------------------------------------------------------------
 
-#External Modules------------------------------------------------------------------------------------
-import sys
+# External Modules----------------------------------------------------------------------------------
 import copy
 from operator import mul
 from functools import reduce
-#External Modules End--------------------------------------------------------------------------------
+# External Modules End------------------------------------------------------------------------------
 
-#Internal Modules------------------------------------------------------------------------------------
+# Internal Modules----------------------------------------------------------------------------------
 from ..utils import InputData, InputTypes
-from .ForwardSampler        import ForwardSampler
+from .Sampler               import Sampler
 from .MonteCarlo            import MonteCarlo
 from .Grid                  import Grid
 from .Stratified            import Stratified
 from .FactorialDesign       import FactorialDesign
 from .ResponseSurfaceDesign import ResponseSurfaceDesign
 from .CustomSampler         import CustomSampler
 from .. import GridEntities
-#Internal Modules End--------------------------------------------------------------------------------
+# Internal Modules End------------------------------------------------------------------------------
 
-class EnsembleForward(ForwardSampler):
+class EnsembleForward(Sampler):
```

Comment:
This change is something that should have been discussed with the development team. Even if it appears there's no mechanical reason for a subclass to exist, it's at least worth checking with the existing team to see if it should be removed.

Reply:
EAFP. I don't like keeping placeholders like this, but if you want it back I can put it back.

Reply:
Probably not worth reverting.